shared/actions/wallets.tsx

```diff
@@ -1027,6 +1027,8 @@ const checkDisclaimer = (_: TypedState, action: WalletsGen.CheckDisclaimerPayloa
           } else {
             actions.push(RouteTreeGen.createNavigateAppend({path: [SettingsConstants.walletsTab]}))
           }
+        } else {
+          actions.push(RouteTreeGen.createNavigateAppend({path: [SettingsConstants.walletsTab, 'wallet']}))
```

Comment:
I found that we need to explicitly navigate to the wallets page now after the disclaimer is accepted as a result of our routing changes.

Reply:
i think if you do a componentDidUpdate call in _OnboardingOrWallets and navigate like didMount did, it'd work?